Cargo.toml

```diff
@@ -3,6 +3,15 @@ name = "ssi"
 version = "0.1.0"
 authors = ["Spruce Systems, Inc."]
 edition = "2018"
+license = "Apache-2.0"
+description = "Core functionality for Verifiable Credentials and Decentralized Identifiers"
```

Comment:
```suggestion
description = "Core library for Verifiable Credentials and Decentralized Identifiers."
```

But I don't have strong feelings about it. Except the full-stop. :stuck_out_tongue:

Reply:
Works for me - thanks!